docs/user-guide/use-certificates.md

```diff
@@ -11,3 +11,17 @@ When Zowe is launched, details for the PKCS12 certificate used are specified in
 The two most common scenarios for using a PKCS12 certtificate are where you have been given an existing certificate and wish to configure Zowe to use it, or else you do not have a certificate and wish to generate a new one.  The `zwe init certificate` command supports both scenarios.  The input parameters that control certificate configuration
 are specified in the section `zowe.setup.certificates`
 
+### Use multiple certificate authorities
+
+If you use mutiple certificate authorites, the syntax in the `zowe.yaml` is shown as below.
+
+```
+certificate:
+  pem:                                                                           
+    key: ""                                                                      
+    certificate: ""                                                              
+    certificateAuthorities:                                                      
+    - "safkeyring:////stcusername/KeyName&ca name 1"        
+    - "safkeyring:////stcusername/KeyName&ca name 2"
+```
+If you run into `<ZWED:527259> SZOWE CRITICAL unable to get issuer certificate`, check this section in your `zowe.yaml` file and see if it is the problem about the syntax.
```

Comment:
@Joe-Winchester Hi Joe, I just added the instructions according to the Gothaer's mail from you. Let me know if you have any comments or questions. Thanks

Reply:
Hi Sam,  this is great to include the syntax for the CAs where there are more than one.  Thank you ! 
It looks like there is a merge conflict with the `sidebars.js`.    It'd be great to have this merged into staging and then I'd love to work more on the chapter content.

Reply:
@Joe-Winchester Hi Joe, thanks for letting me know. Sorry I was on vacation last week and I sent you a message on Slack. I will merge this PR later today, thank you!